src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -341,7 +328,6 @@ public void handleEvent(Event event)
         ds.setTransfer(new Transfer[]{ TextTransfer.getInstance() });
         ds.addDragListener(new DragSourceAdapter()
         {
-            @Override
             public void dragSetData(DragSourceEvent event)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)